l10n-pocket/en/pocket/add.ftl

```diff
@@ -0,0 +1,50 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev-pocket.allizom.org/about/
+
+pocket-how-to-save = How to Save
```

Comment:
Suggestion: every pocket string should have a unique ID, so I'd suggest adding a namespace of `pocket-add-*` to the strings for this page, rather than just `pocket-*`?

Reply:
+1